ravenframework/CodeInterfaceClasses/RELAP5/RELAPparser.py

```diff
@@ -462,12 +479,18 @@ def modifyOrAdd(self,modifyDict,save=True):
       # modify the cards
       for card in cardLines.keys():
         for var in modiDictionaryList[card]:
+          dtype = "float"
+          if card in intDT and int(intDT[card]) == int(var['position']):
```

Comment:
why do we need the second portion of this if condition (int(intDT[card]) == int(var['position']))?

Reply:
yes that is needed because multiple "cards" can be inputted at different position. For example, we can sample 2 cards:  ```201:1``` and ```201:2``` and we want only the second one to be integer. Ie.:
```<integers>201:2</integers>```
If we do not check for the position, both cards will be treated as integers.